src/inverseRobot.js

```diff
@@ -7,7 +7,23 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const first = Object.keys(robot);
+  const second = Object.values(robot);
+  const newObj = {};
+
+  for (let i = 0; i < second.length; i++) {
+    for (let j = i + 1; j < second.length; j++) {
```

Comment:
Simplify the code by using `for ... in` loop
```suggestion
  const newObj = {};
  
  for (const ... in ...) { 
```